docs/en/sql-reference/table-functions/generate_series.md

```diff
@@ -0,0 +1,25 @@
+---
+slug: /en/sql-reference/table-functions/generate_series
+sidebar_position: 146
+sidebar_label: generate_series
+---
+
+# generate_series
+
+`generate_series(START, STOP)` - Returns a table with the single ‘generate_series’ column (UInt64) that contains integers from start to stop inclusively.
+
+`generate_series(START, STOP, STEP)` - Returns a table with the single ‘generate_series’ column (UInt64) that contains integers from start to stop inclusively with spacing between values given by STEP. 
+
+The following queries return tables with the same content but different column names:
+
+``` sql
+SELECT * FROM numbers(10, 5);
+SELECT * FROM generate_series(10, 14);
+```
+
+And the following queries return tables with the same content but different column names (but the second option is more efficient):
```

Comment:
Existing table function `numbers` has overloads
- `numbers(N)` - "generate numbers 0 to N-1"
- `numbers(N, M)` - "generate numbers N to N + M - 1".

Why don't we add a third overload
- `numbers(N, M, S) - "generate numbers N to N + M - 1 with step size S"

? That would imho be more user-friendly than adding a new and unrelated table function.